### ISSUE TITLE
`verdi process ` does not return anything

### ISSUE THREAD

**Issue by agoscinski** [aiidateam/aiida-core#6391]
### Describe the bug / Expected behavior
- No output in `verdi process <SUBCOMMAND>` commands for `show`, `report`, `status`, `watch`

```bash
$ verdi process show
```
There should be some information for the use that the process id is missing. Even more for `watch` it starts to run which I find strange as no id was specified.

- Also there is a inconsistency how the user is informed about missing arguments between `verdi code <SUBCOMMAND>` and `verdi process <SUBCOMMAND>`. Here compare
```bash
$ verdi process kill
Report: no active processes selected.
```
with
```bash
$ verdi code show 
Usage: verdi code show [OPTIONS] CODE
Try 'verdi code show --help' for help.

Error: Missing argument 'CODE'.
```
I would stick to the style of `verdi code show` everywhere, seems to me more informative and the same style is also used in `verdi computer <SUBCOMMAND>`.
That means subcommands `kill`, `pause`, `play`

### Your environment

- Operating system: Ubuntu 22.04 LTS
- Python version: 3.11
```
$ verdi status
✔ version:     AiiDA v2.5.1
✔ config:      /home/alexgo/code/aiida-core/.aiida
✔ profile:     alexgo
✔ storage:     SqliteDosStorage[/home/alexgo/code/aiida-core/.aiida/repository/sqlite_dos_f275ff0f10174e8e84c13e82dd5ba452]: open,
✔ rabbitmq:    Connected to RabbitMQ v3.8.14 as amqp://guest:guest@127.0.0.1:5672?heartbeat=600
```

**Comment by sphuber** [comment]
Fair enough. These commands function in this way because they allow multiple processes to be specified. It simply loops over those, with the result that nothing happens if no processes are specified. Should be easy enough to simply error out if no processes are defined whatsoever.
